README.md

```diff
@@ -59,6 +59,40 @@ Please note:
 * Batch sizes specified in prototxt files are per device.
 * Memory layers should not be shared among GPUs, and thus "share_in_parallel: false" is required for layer configuration.
 
+## Building for Spark 2.X
+To Build for Spark 2.X  please include  
+
+* mvn -Dspark2
+
+**Spark2 Pre-release note**  You will need to do the following:
+- in the Spark git clone :
+
+-      mvn **install** \<other spark options\>
+
+Then:  update the CaffeOnSpark caffe-grid/pom.xml in the "spark2" profile section:  set the correct spark (e.g. **2.0.0-SNAPSHOT**),scala (e.g. **2.11.7**)), and hadoop (e.g. **2.7.1**)) versions:
+
+```
+  <profiles>
```

Comment:
Can't this be added to pom conditionally. Like we have build/buildosx which
do everything required

On Mon, Jun 20, 2016 at 10:13 PM, StephenBoesch <notifications@github.com>
wrote:

> In README.md
> <https://github.com/yahoo/CaffeOnSpark/pull/79#discussion_r67809032>:
>
> > @@ -59,6 +59,40 @@ Please note:
> >  * Batch sizes specified in prototxt files are per device.
> >  * Memory layers should not be shared among GPUs, and thus "share_in_parallel: false" is required for layer configuration.
> >
> > +## Building for Spark 2.X
> > +To Build for Spark 2.X  please include
> > +
> > +* mvn -Dspark2
> > +
> > +**Spark2 Pre-release note**  You will need to do the following:
> > +- in the Spark git clone :
> > +
> > +-      mvn **install** \<other spark options\>
> > +
> > +Then:  update the CaffeOnSpark caffe-grid/pom.xml in the "spark2" profile section:  set the correct spark (e.g. **2.0.0-SNAPSHOT**),scala (e.g. **2.11.7**)), and hadoop (e.g. **2.7.1**)) versions:
> > +
> > +```
> > +  <profiles>
>
> I had already done so. But then @mriduljain
> <https://github.com/mriduljain> did not even "notice" / see the
> instructions. These instructions make it more clear. If you feel strongly
> then bring it up with him since I added this based on his feedback.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/yahoo/CaffeOnSpark/pull/79/files/0a3a9daf07ae0b44a3f2e63a272f631b1130f937#r67809032>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACCTVbhB1x5tnhKgH9ty1mdlfQ5VMkhEks5qN3L0gaJpZM4IxfyE>
> .
>